tasks/main.yml

```diff
@@ -133,6 +133,14 @@
        virtualenv_python=/usr/bin/python{{ python_version }}
```

Comment:
Two lines up from here: the way we're doing this only works if the requirements file is in a subdirectory called requirements.  Not all projects do that: I just tested with https://github.com/caktus/taytay, for example, which just put its requirements files in its top-level source dir.  Let's let people specify the full path into the source tree.